docs/build/guides/conventions/soroban-contract-init-template.mdx

```diff
@@ -0,0 +1,339 @@
+
+```json
+"scripts": {
+    "setup": "node initialize.js",
+  },
+
+ "workspace": ["./packages/*"],
+```
+
+## Running the script
+
+Now that we have a complete script and a frontend, we should try to run the script. However, because the script is dependent on contract files we need to have a contract project ready, otherwise the script will fail and result in an error. To solve this issue we have to either copy the whole project contents and paste them to an existing Stellar contract project directory or upload the template file to a repo then create a new project and setting the 'f' parameter to point to the hosted repository of the template.
+
+### Setting up the project
+
+I have hosted a [public repository of the React template](https://github.com/CaptainPrinz/soroban-react-template). The following snippet implements the second option:
+
+```bash
+stellar contract init token_contract -w token -f https://github.com/CaptainPrinz/soroban-react-template.git
+```
+
+The snippet above initializes a new Soroban project, `token_contract` that uses a prebuild token contract and adds the frontend template we deployed to a remote repository.
+
+### Setting the environment variables
+
+Our templates offer a great deal of customization via environment variables. These variables need to be set by the user by creating a `.env` file otherwise the process will fail as these variables will be undefined.
+
+### Starting the script
+
+On your terminal run the following code:
+
+```bash
+npm run setup
+```
+
+This runs the `initialize.js` script and first sets the network profile, account identities, builds the contract `.wasm` file, deploys the contracts, and generates the bindings.
+
+If successful, there should be the following changes in the directory:
+
+1. A new `.soroban` directory, which contains the network profile files, account identity files, and hosted smart contract files in json format
+2. A `package` directory containing the generated TypeScript bindings
+3. A `contracts.json` file
+
+You are now finally able to import these generated bindings in your frontend application, create their clients, and use them to interact with the contracts deployed on the network.
+
+## Customization
+
+With the `initialize.js` script being the heart of the template file, you can go through the file and modify the paths in which files are saved in if necessary.
+
+Also a great deal of customization is enabled by using environment variables, giving flexibility to change network profiles and account identity without limits.
+
+## Conclusion
+
+This section concludes the guide. Check out the documentation on how bindings can be used in dappa as this is a template tutorial is limited to areas concerned with templates as such the processes of connecting these generated bindings and using them in frontend appications is not covered.
```

Comment:
```suggestion
This section concludes the guide. Check out the documentation on how bindings can be used in dapps as this is a template tutorial is limited to areas concerned with templates as such the processes of connecting these generated bindings and using them in frontend applications is not covered.
```
Couple spelling errors

Reply:
This sentence is also a bit of a run-on. Can you clean up or rephrase this to be a little clearer?